src/angulartics-tealium.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license Angulartics v0.17.2
+ * (c) 2013 Luis Farzati http://luisfarzati.github.io/angulartics
+ * Tealium Plugin Contributed by https://github.com/ianhampton & https://github.com/simonbrowning
+ * License: MIT
+ */
+
+(function(angular){
+'use strict';
+
+
+/**
+ * @ngdoc overview
+ * @name angulartics.tealium
+ * Enables angulartics native integration with the Tealium utag.js
+ */
+
+angular.module('angulartics.tealium', ['angulartics'])
+.config(['$analyticsProvider', function($analyticsProvider){
+  var dataCheck;
+  
+  /**
+  * dataCheck function to convert data sources into Tealium proposed standard. i.e. 'page_name'.
+  */
+
+  dataCheck = function(data){
```

Comment:
convertHashToSnakeCase or something similar looks like a better name. What do you think?